release-notes/opensearch-asynchronous-search.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,21 @@
+## Version 1.2.0.0 2021-11-05
+
+Compatible with OpenSearch 1.2.0
+
+### Bug Fixes
+
+* Add checkstyle plugin reference [#60](https://github.com/opensearch-project/asynchronous-search/pull/60)
```

Comment:
Please enclose PRs in rounded brackets ([#60](https://github.com/opensearch-project/asynchronous-search/pull/60))